src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -545,6 +545,11 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
       ResourceName="XA1035"
       Condition=" '$(BundleAssemblies)' == 'true' and '$(UsingAndroidNETSdk)' == 'true' "
   />
+  <AndroidError Code="XA1038"
+      ResourceName="XA1038"
+      Condition=" '$(_AndroidWarnOnTargetPlatformVersion)' != '' "
+      FormatArguments="$(_AndroidWarnOnTargetPlatformVersion);$(TargetFramework);$(_AndroidLatestStableApiLevel)"
```

Comment:
I don't quite understand `dotnet build` handling of `$(TargetFramework)`:

Assume a project with `$(TargetFramework)`=net8.0-android33.0.

By the time we encounter *this* line, is `$(TargetFramework)` *changed* to `net8.0-android`, moving the `33.0` into `$(TargetFrameworkVersion)`?  Or does it remain `net8.0-android33.0`?

I'm curious because of the error message text: if it *isn't* changed to remove the `33.0`, we'd have an error message such as:

```
$(TargetPlatformVersion) 33.0 is not a valid target for 'net8.0-android33.0 projects. Please update your $(TargetPlatformVersion) to a supported version (e.g. 34).
```

Reply:
This is ugly, but I think the best option is to use `net$(TargetFrameworkVersion.TrimStart('vV'))`, so it will be:

    $(TargetPlatformVersion) 33.0 is not a valid target for 'net8.0' projects. Please update your $(TargetPlatformVersion) to a supported version (e.g. 34).